terraform/main.tf

```diff
@@ -69,6 +69,18 @@ module "website" {
   origin_artifacts_dist_path = coalesce(
     var.website_origin_artifacts_dist_path, "${path.root}/../packages/client/dist"
   )
+
+  datadog_rum_enabled = var.website_datadog_rum_enabled
+  datadog_rum_config = merge(var.website_datadog_rum_options, {
+    applicationId       = "15db471e-2ccb-4d3c-a6bf-99b750d748f5"
+    clientToken         = "pub50834fcc1999d53e546519b1a0f03934"
```

Comment:
Just to be clear: these values are _not_ considered secret and are fine to be committed.